README.md

```diff
@@ -32,16 +32,17 @@ spec:
       jaeger:
 
     processors:
-      queued-retry:
+      queued_retry:
```

Comment:
@pjanotti, could you please review the changes to this yaml file? This seems required to migrate from 0.0.2 to 0.2.0.